docs/test/Runtime 0.1.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7f409f1f",
+   "metadata": {},
+   "source": [
+    "# Acceptance Criteria for Runtime 0.1 release"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d039414",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import IBMRuntimeService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7300dad",
+   "metadata": {},
+   "source": [
+    "# Authentication\n",
+    "\n",
+    "## Legacy\n",
+    "Using IBM Quantum API token.\n",
+    "\n",
+    "### Save Legacy Account"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0326e78b-b3df-40ad-9adb-db7a03a29f9c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save default legacy account\n",
+    "IBMRuntimeService.save_account(\n",
+    "    auth=\"legacy\", \n",
+    "    token=\"xyz\", \n",
+    "    url=\"https://auth.quantum-computing.ibm.com/api\", \n",
+    "    instance=\"qiskit-hub/qiskit-group/qiskit-project\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ca5c496d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save legacy account with a name\n",
+    "IBMRuntimeService.save_account(\n",
+    "    name=\"my-legacy-account\"\n",
+    "    auth=\"legacy\", \n",
+    "    token=\"abc\", \n",
+    "    url=\"https://auth.quantum-computing.ibm.com/api\", \n",
+    "    instance=\"qiskit-hub-1/qiskit-group-1/qiskit-project-1\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5232022a",
+   "metadata": {},
+   "source": [
+    "### Load Legacy Account from Saved Account"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f45dccc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load default legacy account\n",
+    "service = IBMRuntimeService(auth=\"legacy\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bfcc2cec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load legacy account with name\n",
+    "service = IBMRuntimeService(name=\"my-legacy-account\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76d4c05a",
+   "metadata": {},
+   "source": [
+    "### Enable Legacy Account for current session without saving"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ec0980d9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enable legacy account with auth and token parameters\n",
+    "service = IBMRuntimeService(auth=\"legacy\", token=\"xyz\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f76d6841",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enable legacy account with token and url parameters\n",
+    "service = IBMRuntimeService(token=\"xyz\", url=\"https://auth.quantum-computing.ibm.com/api\")"
```

Comment:
Based on the latest [implementation](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/ibm_runtime_service.py#L228-L249) from @jyu00 this needs to be changed. 

```python
# ...
# Let's not infer based on these attributes as they may change in the future.
raise ValueError(
  "'auth' is required if 'token', or 'url' is specified but 'name' is not."
)
# ...
```

Reply:
Yeah I'd prefer _not_ infer the auth type based on url, especially since we may have additional cloud provider in the future.